eng/scripts/Verify-And-Merge-PRs.ps1

```diff
@@ -1,111 +1,132 @@
 param(
-  $PRDataArtifactPath,
-  $AuthToken,
-  $ShouldMerge
+  [string]$PRDataArtifactPath,
+  # @('Azure;azure-sdk-for-net;1', ...)
+  [array]$PRDataInline,
```

Comment:
Did you just this just for local testing?

Reply:
Yep it's just for local. I can scrap it, but I imagine there will be some hardening so I think it's useful to keep local hooks in for now.